dateparser_data/supplementary_language_data/date_translation_data/en.yaml

```diff
@@ -47,12 +54,30 @@ relative-type:
         - this decade
     in 1 decade:
         - next decade
+    1 century ago:
+        - last century
+        - this century
+    in 1 century:
+        - next century
+    1 fortnight ago:
+        - last fortnight
+        - this fortnight
+    in 1 fortnight:
+        - next fortnight
 
 relative-type-regex:
     in \1 decade:
         - in (\d+) decades?
     \1 decade ago:
         - (\d+) decades? ago
+    in \1 century:
+        - in (\d+) century?
+    \1 century ago:
+        - (\d+) century? ago
+    in \1 fortnight:
+        - in (\d+) fortnight?
+    \1 fortnight ago:
+        - (\d+) fortnight? ago
```

Comment:
This is not correct. The `?` indicates an optional letter. 

Doing this it will accepts "in 1 century" but also "in 1 centur". You need to add an `s` before the `?`.
Also, you should need to add support for `centuries` here.

You can check if both thigns work with this
```python
 dateparser.parse("in 3 centurys")
```
and:
```python
 dateparser.parse("in 3 centuries")
```
Now both are failing.

Reply:
@noviluni sure fixing the bug.